doc/classes/Crypto.xml

```diff
@@ -1,8 +1,27 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Crypto" inherits="Reference" category="Core" version="3.2">
 	<brief_description>
+		Access to advanced cryptographic functionalities.
 	</brief_description>
 	<description>
+		The Crypto class allows you to access some more advanced cryptographic functionalities in Godot.
+		For now, this includes generating cryptographically secure random bytes, and RSA keys and self-signed X509 certificates generation. More functionalities are planned for future releases.
+		[codeblock]
+		extends Node
+
+		var _crypto = Crypto.new()
```

Comment:
We don't prefix member variables with `_` in the https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html, so it would be best not to do it here either. This would also be consistent with the description of `generate_self_signed_certificate`.